db/schema.rb

```diff
@@ -43,6 +43,7 @@
     t.integer  "customer_id"
     t.integer  "c2_status",       default: 0,    null: false
     t.string   "token"
+    t.datetime "archived_at"
```

Comment:
I understand, but I would like if we document these things in the issue formally since I missed the grooming and find myself implementing things we don't need :(